packages/material-ui-utils/src/getScrollbarSize.ts

```diff
@@ -1,16 +1,7 @@
 // A change of the browser zoom change the scrollbar size.
-// Credit https://github.com/twbs/bootstrap/blob/3ffe3a5d82f6f561b82ff78d82b32a7d14aed558/js/src/modal.js#L512-L519
+// Credit https://github.com/twbs/bootstrap/blob/488fd8afc535ca3a6ad4dc581f5e89217b6a36ac/js/src/util/scrollbar.js#L14-L18
 export default function getScrollbarSize(doc: Document): number {
```

Comment:
AFAIK we can't modify this helper. This one is meant to measure the size of the scrollbar, no matter if there is one or not. The new logic only measures the size of the scrollbar, if there is one. (`window.innerWidth - document.documentElement.clientWidth` returns 0).

Reply:
But maybe we should make it deprecated. I mean we have `ScrollbarSize` that seems a solid alternative.

Reply:
This helper is only used in ModalManager and MenuList. ModalManager checks if scrollbar is present beforehand, so there's no issue there. As for MenuList: I don't know what is its purpose there - it seems to work the same with the new logic in [the Max height menu example](https://deploy-preview-26816--material-ui.netlify.app/components/menus/#max-height-menu).

Reply:
My concern is more with the public API. This is a breaking change. We were using it in @mui-org/x until recently: https://github.com/mui-org/material-ui-x/pull/1703. 

Oh no, my bad. It's exposed as `unstyled_`. Great, we can move forward.